react-carousel-component/my-app/src/IndexButtons.js

```diff
@@ -0,0 +1,19 @@
+import './IndexButtons.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircle as solid } from '@fortawesome/free-solid-svg-icons';
+import { faCircle as regular } from '@fortawesome/free-regular-svg-icons';
+
+export default function IndexButtons({ items, onIndex, index }) {
+  return (
+    <div>
+      {items.map((item) => (
+        <FontAwesomeIcon
+          icon={index === items.indexOf(item) ? solid : regular}
```

Comment:
This check isn't reliable since you could conceivably have the same value of `item` in multiple locations in the array. The best way to do it is using the index that you get from `items.map`:
```
items.map((item, idx) => ... icon={index === idx ? solid : regular} ...
```